codec-ntg/README.md

```diff
@@ -0,0 +1,13 @@
+# NTG codec
+
+This codec can be used for decoding and encoding message in NTG (Native) protocol.
+
+## Configuration
+
+### Codec factory
+
+To use the following codec you need to specify the following codec factory:
```

Comment:
To use the **subsequent** codec **,** you **will** need to specify the following codec factory: